tests/UI/pages/FO/searchResults/index.js

```diff
@@ -89,6 +90,15 @@ class SearchResults extends FOBasePage {
 
     return this.getAttributeContent(page, this.quickViewCoverImage, 'src');
   }
+
+  /**
+   * Get the product price value
+   * @param page
+   * @returns {Promise<string>}
+   */
+  async getProductPrice(page) {
+    return this.getTextContent(page, this.productPrice);
+  }
```

Comment:
FYI : if there is no await on the function, you don't need the `async` in function definition. 

```suggestion
  getProductPrice(page) {
    return this.getTextContent(page, this.productPrice);
  }
```